tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "strict": true,
     "strictFunctionTypes": false,
     "skipLibCheck": true,
-    "sourceMap": true,
+    // temporary workaround @see https://github.com/cypress-io/cypress/issues/26203#issuecomment-2152558848
+    "sourceMap": false,
```

Comment:
I don't like the fact that including `vueUse` library required to raise the TS version, the testing framework versions and led to this source map disabling.
If the vueUse lib was included only to open the file upload window I think it woud;d be better to have our own tiny util function for this.

What do you think @CedrikNikita ?

Reply:
Raising TS version suppose to be heavily tested i guess. I guess it is ok to check custom function if it proves to be more efficient and maintainable. Would you be interested in collaborating on a solution?"

Reply:
I suppose you're right